test/test_torch.py

```diff
@@ -2547,13 +2537,13 @@ def test_ones_like_multiple_device(self):
         self.assertEqual(output, expected)
 
     def test_dtypes(self):
-        all_dtypes = torch.testing.get_all_dtypes()
+        all_dtypes = torch.testing.get_all_dtypes(True)
```

Comment:
This is giving me severe boolean blindness. What is `True` here supposed to mean? I don't know unless if I look at the definition of `get_all_dtypes`. It would be much better if you explicitly specified a keyword argument here, making its meaning clear at first glance.

Reply:
I'd suggest the following:

you are typically trying to differentiate between "all" types and "types with math."  This is dependent on the device.  So just have two functions.